mbbsd/bbslua.c

```diff
@@ -977,7 +977,9 @@ bbsluaRegConst(lua_State *L)
 
     // unbind unsafe API
```

Comment:
Please help add following comment:
// Loading bytecode is dangerous. Lua 5.1 has bytecode verifier but doesn't guarantee safe. Lua 5.2 even removed the verifier.